ui/packages/ace/lib/start.js

```diff
@@ -1,14 +1,19 @@
 const open = require('opn')
 const webpack = require('webpack')
 const WebpackDevServer = require('webpack-dev-server')
+const clc = require('cli-color')
 
 const webpackConfig = require('./webpack.config')
+const warn = clc.yellowBright.blink
```

Comment:
`yellowBright` corresponds to the color code for bright yellow as opposed to "bright/bold" + "yellow". The bright yellow color code is not widely used and not supported by some terminals; `yellow.bold` is more commonly used and better supported.
`blink` is rarely supported and would be rather annoying even if it is supported.